AzureTable/Microsoft.DataTransfer.AzureTable/RemoteLogging/LoggingTableEntity.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Azure.CosmosDB.Table;
+
+namespace Microsoft.DataTransfer.AzureTable.RemoteLogging
+{
+    /// <summary>
+    /// Class to define the schema for Logging data being sent to the table
+    /// </summary>
+    public class LoggingTableEntity : TableEntity
+    {
+
```

Comment:
Remove empty line?

Reply:
N/A here .. applied feedback to the interface in new location Core project.